contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -3388,7 +3387,14 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 
 					bbf_ExecDropStmt(drop_stmt);
 					del_ns_ext_info(schemaname, drop_stmt->missing_ok);
-					clean_up_bbf_schema_permissions(logicalschema, NULL, true);
+					if (strcmp(queryString, "(DROP DATABASE )") != 0)
```

Comment:
might improve some query performance by evaluating `strcmp(queryString, "(DROP DATABASE )") != 0` once and reusing it at L3376 and L3390

Reply:
Addressed.